HowToRPC.md

```diff
@@ -0,0 +1,166 @@
+# How to Call Google APIs: RPC Edition
+
+_For many developers, the easiest way to call a Google API is with one of our
+client libraries. But occasionally someone may prefer to make API calls
```

Comment:
someone => one

Reply:
Could you explain why you prefer "one"? 

Reviewing the comments and the answer to [this question comparing one vs. someone](https://english.stackexchange.com/questions/120592/one-vs-someone-can-be-used-interchangeably), I think "someone" is a better fit: by indicating a "generic specific" individual, it implies that directly making API calls is a less-common situation. 

If we said "one" it would seem like we might be [circumspectly/euphemistically referring to ourselves or the reader](https://english.stackexchange.com/questions/120592/one-vs-someone-can-be-used-interchangeably#comment247695_120592), which would seem like a hint that this is actually a preferred practice and not a fallback.

Reply:
Considering we use "you" two lines later, it is fine to use it here also.